src/pybind/mgr/volumes/fs/stats_util.py

```diff
@@ -177,7 +177,7 @@ def _get_clone_dst_info(self, fs_handle, ci, clone_entry,
             ci.dst_path = dst_subvol.path
             log.debug(f'destination subvolume path for clone - {ci.dst_path}')
 
-        clone_state = get_clone_state(self.volclient, self.vol_spec, ci.volname,
+        clone_state = get_clone_state(self.volclient, self.volspec, ci.volname,
```

Comment:
In L#132, its `self.vol_spec = vol_spec`. So, why change this? 🤔

Reply:
Fixed.